stable/nginx/nginx/values.yaml

```
@@ -22,9 +22,14 @@ Ingress:
   # The class of the ingress controller to use. 
   # For SLATE this should be 'slate'. 
   Class: slate
-  
+
+#Defines what IP ranges can access the app
+#If disabled all IP addresses can access the app
+NetworkPolicy:
+  Enabled: false
+  AllowedCIDR: 10.0.0.0/255
```

```suggestion
  AllowedCIDR: 10.0.0.0/8
```

This is not a valid CIDR.